roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,7 +209,7 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-metadata-agent', 'neutron-linuxbridge-agent']
```

Comment:
I'd break this up to use multiple lines re: pep8's indentation standards: https://www.python.org/dev/peps/pep-0008/#indentation

additionally, consider using a tuple for sequences that will not be mutated, they are considerably faster in the VM.
https://stackoverflow.com/questions/68630/are-tuples-more-efficient-than-lists-in-python